proposals/lip-maxime-vote_by_locking.md

```diff
@@ -0,0 +1,491 @@
+
+
+## Rationale
+
+It is important to acknowledge that one perfect voting system for Delegated Proof of Stake does not exist. Nevertheless, we believe that the proposed change of the voting system is a significant improvement over the current state and an important step in evolving the Lisk Delegated Proof of Stake system.
+
+
+### Technical Glossary
+
+
+
+*   Delegate weight: the weight of the delegate in the process of choosing the block forgers. The way this weight is calculated is described in the 
+[specification section](#specification). This proposal does not modify the way we use the delegate weights to select forging delegates.
+*   Voting: the way LSK holders use to show support for a delegate. Users can vote for delegates and those votes impact the delegate weight.
+*   Unvoting: the action of removing one’s votes.
+*   Self-voting: delegates can vote for themselves, this action is called self-voting.
+*   Locked tokens: tokens that cannot be used. Such tokens have to be unlocked (and potentially wait a given number of blocks) before being available again. They do not leave the account of the holder, but their use is invalid.
+*   Unlocking: process of getting locked LSK tokens available again.
+*   Balance: amount of available LSK in an account. Notice that the total owned by an account is therefore “balance + locked tokens”.
+
+
+### Desired Properties
+
+At first, we want to list the main desired properties of a voting system for Lisk. The following properties are not listed in order of importance.
+
+
+
+*   **Decentralization:** In order to have a truly decentralized network, the delegates should act as independent entities and the voting system should give no incentive for delegates to form coalitions. Instead, it should foster competition for the active delegate ranks. The future implementation of the [BFT consensus protocol](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md) will provide block finality to the Lisk protocol. However, it assumes that there are at least 68 delegates honestly following the protocol. That is a further reason to encourage influence groups or coalitions to remain small and independent.
+*   **Flexibility:** The voting system should allow users to vote for different delegates and to tune their votes to suitably reflect their voting preferences. In turn, this encourages the participation of stakeholders in the voting system.
+*   **Security:** The voting system should incentivize voters to elect delegates who maintain a secure and effective network.
+*   **Efficiency:** The voting system should allow to efficiently compute the rank of the delegates. Moreover, vote transactions should be kept small.
+
+Another property that may be intuitively named is “fairness”. This property is very subjective and is hard to precisely define, but we try to address it in the next subsection. 
+
+
+### One Voting Lisk – 
+Power
```

Comment:
I don't really understand this title. Before it was `One Voting Lisk – One Vote Power`.

Reply:
Changed back.